src/screens/ContactScreen/index.js

```diff
@@ -4,7 +4,7 @@ import {DataProvider, LayoutProvider, RecyclerListView} from 'recyclerlistview';
 import {Dimensions, RefreshControl, SafeAreaView, StatusBar, StyleSheet, View} from 'react-native';
 /* eslint-disable no-param-reassign */
 import {debounce} from 'lodash';
-
+import {useRoute} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useRoute` from `@react-navigation/core` is a new addition. This hook allows access to the route prop in any component. It's a good practice to use this hook when you need to access route params outside of screen components.
```diff
+ import {useRoute} from '@react-navigation/core';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->